### PR TITLE
fix: Improve prerender errors

### PR DESCRIPTION
**Pull request by rschristian**
**What kind of change does this PR introduce?**

Bugfix / revert

**Did you add tests for your changes?**

N/A

**Summary**

TLDR:

Before PR               | Improved Error Logic | Improved Error Logic & Disabled SSR Build Minification
:-------------------------:|:-------------------------:|:-------------------------:
Before this PR | With Error Logic Altered | Disabled SSR Build Minification

Partial revert of #1426

Having used it for a bit, I don't find it to be particularly compelling, especially w/ Webpack v5 -- `methodName` is often null and not correct enough for determining user vs lib code.

Additionally, I've found the stack trace locations (from Node / v8, to be clear) to be exceptionally poor on Webpack's minimized output, and so have elected to not minimize our SSR build.

Take the following:

> src/routes/home/index.js

```jsx
import { h } from 'preact';
import style from './style.css';

console.log(foo.bar);

const Home = () => (
	<div class={style.home}>
		<h1>Home</h1>
		<p>This is the Home component.</p>
	</div>
);

export default Home;
```

As `foo` is not defined, we should get a (reference) error here (and we do). However, the stack trace of `ssr-bundle.js` actually points to the `h("div", { class: ... }, ...)`.

Why this is, I don't know. Mix of Webpack's output and Node maybe? Regardless, it's not great and reduces the quality of our error reporting (as it highlights the wrong line). But what we can do is just not minimize the SSR output. As it's only ran in Node, we don't have any particular reason _to minimize_, and a couple against, such as:
  - faster builds of the SSR bundles as minification takes time
  - better error positions if prerender errors occur

**Does this PR introduce a breaking change?**

No

### CODE REVIEW

File: packages/cli/src/lib/webpack/prerender.js [hunk 1]
```diff
@@ -45,112 +45,98 @@ async function handlePrerenderError(err, env, stack, entry) {
...
-					let line = position.line + i;
-					let sourceLine = sourceLines[line - 1];
-					sourceCodeHighlight += sourceLine ? `${color(sourceLine)}\n` : '';
-				}
+		if (sourceLines) {
+			for (var i = -4; i <= 4; i++) {
+				let color = i === 0 ? red : yellow;
+				let line = position.line + i;
+				let sourceLine = sourceLines[line - 1];
+				sourceCodeHighlight += sourceLine ? `${color(sourceLine)}\n` : '\n';
```
Comment (rschristian) [inline packages/cli/src/lib/webpack/prerender.js:95]: Rather importantly, we weren't handling blank lines correctly as the ternary was falling back to `''`, instead of the new line as it does above. This caused further issues in line positioning.
